extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResourceImpl.java

```diff
@@ -163,4 +174,12 @@ public TransferHistoryPage getTransferHistoryPage() {
     public UiAsset getTransferProcessAsset(String transferProcessId) {
         return transferHistoryPageAssetFetcherService.getAssetForTransferHistoryPage(transferProcessId);
     }
+
+    private void validate(Object object) {
+        val validator = factory.getValidator();
+        val constraintViolations = validator.validate(object);
+        if (!constraintViolations.isEmpty()) {
+            throw new ConstraintViolationException("Failed to validate", constraintViolations);
+        }
+    }
```

Comment:
- would advocate for a separate unit (separation of concerns). This class is already hopelessly overloaded, it does not need any other concern
- "Validator" is always taken, so add some flaire to it, e.g. "ValidatorUtils" idk
- "ValidatorUtils.validateObj(Object)" can be there